more

```diff
@@ -0,0 +1,13 @@
+./commands/property.go:		// get [1;36;40m[Kcurrent[m[K props
```

Comment:
delete this file?

Reply:
all better... was seeing what would happen if client limit value exceeded server-side...